x-pack/test/security_solution_cypress/cypress/tasks/create_new_rule.ts

```diff
@@ -126,11 +127,15 @@ import { waitForAlerts } from './alerts';
 import { refreshPage } from './security_header';
 import { EMPTY_ALERT_TABLE } from '../screens/alerts';
 
+export const createAndEnableRuleOnly = () => {
+  cy.get(CREATE_AND_ENABLE_BTN).click({ force: true });
+  cy.get(CREATE_AND_ENABLE_BTN).should('not.exist');
+};
```

Comment:
Posted a comment on the `createAndEnableRule` function in https://github.com/elastic/kibana/pull/166767#discussion_r1330689150 - I guess you had to create this new function because the existing one was doing navigation using breadcrumbs.